### PR TITLE
feat(parseXML.js): Generate data v3

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Output v3 format data to the console. This PR is for reference only. **Do not merge.**

## Related Issue

- https://github.com/hal-shu-sato/apm-data/pull/144

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/lib/parseXML.js [hunk 1]
```diff
@@ -287,6 +349,57 @@ class PackagesList extends Object {
...
+                  delete release.archiveIntegrity;
+                }
+              }
+            }
+            p.downloadURLs = [p.downloadURL];
+            delete p.downloadURL;
+            if (p.downloadMirrorURL) {
+              p.downloadURLs.push(p.downloadMirrorURL);
+              delete p.downloadMirrorURL;
+            }
```
Comment (hal-shu-sato) [inline src/lib/parseXML.js:395]:
```suggestion
            p.downloadURL = [p.downloadURL];
            if (p.downloadMirrorURL) {
              p.downloadURL.push(p.downloadMirrorURL);
              delete p.downloadMirrorURL;
            }
            p = JSON.parse(
              JSON.stringify(p).replaceAll('downloadURL', 'downloadURLs')
            );
```

**Review by hal-shu-sato — COMMENTED** [review]
I just send the idea of team-apm/apm-data#159.

File: src/lib/parseXML.js (hunk 1 above)
Comment (mitosagi) [reply src/lib/parseXML.js:395]: Thank you.

**Comment by mitosagi** [comment]
Data conversion has been implemented in https://github.com/team-apm/apm-data/pull/190 .

**Pull request closed without merging**